libcontainer/container.go

```diff
@@ -43,7 +49,7 @@ func (s Status) String() string {
 	case Destroyed:
 		return "destroyed"
 	default:
-		return "undefined"
+		panic(fmt.Sprintf("Should never get here: %#v", s))
```

Comment:
I was thinking about removing the entire "default" case but wasn't sure if we needed to worry about some other random value showing up.  If you're ok with me removing it then I will.